.github/workflows/deploy_container_image.yml

```diff
@@ -83,10 +92,11 @@ jobs:
         with:
           src: ${{ inputs.image }}
           dst: |
-            ${{ steps.aws-config.outputs.aws-account-id }}.dkr.ecr.eu-west-1.amazonaws.com/flowforge/${{ inputs.service_name }}:nightly
+            ${{ steps.aws-config.outputs.aws-account-id }}.dkr.ecr.eu-west-1.amazonaws.com/flowforge/${{ inputs.service_name }}:nightly${{ inputs.image_tag_prefix }}
```

Comment:
For staging we have the stacks setup to follow the `nightly` tag so anything newly created or suspended/restarted always picks up the latest build of the NR container.

This will stop that happening automatically unless we also tag them with `nightly` as well.

Reply:
Adjusted according to the discussion.